portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,119 @@
 
 "use strict"
 
+// is called when page loads
+document.addEventListener("DOMContentLoaded", () => {
+    // set max-comments variable
+    const DEFAULT_COMMENTS_NUMBER = 3;
+    const MAX_COMMENTS_NUMBER = 10;
+    let maxComments = DEFAULT_COMMENTS_NUMBER;
+    // set event to change maxComments variable
+    const maxCommentsElement = document.getElementById("max-comments");
```

Comment:
Would suggest to space your code out a bit. It would be easier to read. 

You could add a line every now and then, wherever you're done with a small logical flow.